### ISSUE TITLE
Node Error: Transaction expired

### ISSUE THREAD

**Issue by pedrofaustino** [iron-fish/ironfish#652]
Current version, installed from Dec. 04: `0.1.8 @ src`. My graffiti is `nos-postulo-secretum`

I found the error below when trying to understand the status by running `ironfish status -f`, which printed `Node: STOPPED`.

Below some outputs that might be helpful.

Output from `journalctl -u ironfishd`:
```
Dec 07 17:28:12 machine-id yarn[727406]: Added block seq: 12720, hash: 00000...d25d4, txs: 2, progress: 100.00%, time: 5491.4ms
Dec 07 17:51:42 machine-id yarn[727406]: Added block seq: 12740, hash: 00000...d5b5b, txs: 2, progress: 100.00%, time: 5635.5ms
Dec 07 17:59:44 machine-id yarn[727406]: Added block to fork seq: 12744, head-seq: 12744, hash: 00000...f0c41, head-hash: 00000...25b0c, work: 741769492005294, h>
Dec 07 18:17:30 machine-id yarn[727406]: Added block seq: 12760, hash: 00000...ad2a2, txs: 1, progress: 100.00%, time: 57.2ms
Dec 07 18:18:25 machine-id yarn[727406]: /root/ironfish/ironfish/build/src/blockchain/blockchain.js:569
Dec 07 18:18:25 machine-id yarn[727406]:                     throw new Error(verification.reason);
Dec 07 18:18:25 machine-id yarn[727406]:                           ^
Dec 07 18:18:25 machine-id yarn[727406]: Error: Transaction expired
Dec 07 18:18:25 machine-id yarn[727406]:     at /root/ironfish/ironfish/build/src/blockchain/blockchain.js:569:27
Dec 07 18:18:25 machine-id yarn[727406]:     at async LevelupDatabase.withTransaction (/root/ironfish/ironfish/build/src/storage/database/database.js:39:28)
Dec 07 18:18:25 machine-id yarn[727406]:     at async Blockchain.newBlock (/root/ironfish/ironfish/build/src/blockchain/blockchain.js:519:16)
Dec 07 18:18:25 machine-id yarn[727406]:     at async MiningDirector.constructAndMineBlock (/root/ironfish/ironfish/build/src/mining/director.js:257:26)
Dec 07 18:18:25 machine-id yarn[727406]:     at async MiningDirector.constructAndMineBlockWithRetry (/root/ironfish/ironfish/build/src/mining/director.js:200:26)
Dec 07 18:18:40 machine-id yarn[727387]: error Command failed with exit code 1.
Dec 07 18:18:40 machine-id yarn[727387]: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Output from running `ps -fax`:
```
 727338 ?        Ssl    4:09 node /usr/bin/yarn --cwd /root/ironfish/ironfish-cli/ start start
 727358 ?        S      0:00  \_ /bin/sh -c node start start
 727359 ?        Sl     0:00      \_ /usr/bin/node start start
 727366 ?        S      0:00          \_ /bin/sh -c tsc-watch --build --onSuccess "yarn run start:js start"
 727367 ?        Sl     0:00              \_ /usr/bin/node /root/ironfish/ironfish-cli/node_modules/.bin/tsc-watch --build --onSuccess yarn run start:js start
 727376 ?        Sl   100:52                  \_ /usr/bin/node /root/ironfish/node_modules/typescript/bin/tsc --build --watch
1013263 ?        Ssl    1:20 node /usr/bin/yarn --cwd /root/ironfish/ironfish-cli/ start miners:start -t 4
1013280 ?        S      0:00  \_ /bin/sh -c node start miners:start -t 4
1013281 ?        Sl     0:00      \_ /usr/bin/node start miners:start -t 4
1013288 ?        S      0:00          \_ /bin/sh -c tsc-watch --build --onSuccess "yarn run start:js miners:start -t 4"
1013289 ?        Sl     0:00              \_ /usr/bin/node /root/ironfish/ironfish-cli/node_modules/.bin/tsc-watch --build --onSuccess yarn run start:js miners:start -t 4
1013298 ?        Sl    48:13                  \_ /usr/bin/node /root/ironfish/node_modules/typescript/bin/tsc --build --watch
1170999 ?        Ss     0:00 /lib/systemd/systemd --user
1171002 ?        S      0:00  \_ (sd-pam)
```

**Comment by NullSoldier** [comment]
Here is the error without any of the extra logging. As a note, too many users are reporting this issue where mined blocks are being rejected because of a double spend. We should investigate it.

```
Error: Transaction expired
    at /usr/src/app/node_modules/ironfish/build/src/blockchain/blockchain.js:569:27
    at async LevelupDatabase.withTransaction (/usr/src/app/node_modules/ironfish/build/src/storage/database/database.js:39:28)
    at async Blockchain.newBlock (/usr/src/app/node_modules/ironfish/build/src/blockchain/blockchain.js:519:16)
    at async MiningDirector.constructAndMineBlock (/usr/src/app/node_modules/ironfish/build/src/mining/director.js:257:26)
    at async MiningDirector.constructAndMineBlockWithRetry (/usr/src/app/node_modules/ironfish/build/src/mining/director.js:200:26)
```

**Comment by Hugoo** [comment]
Same issue here:

```
Added block to fork seq: 16241, head-seq: 16241, hash: 00000...29b0f, head-hash: 00000...257de, work: 1240923492388141, head-work: 1240923492388141, work-diff: 0
/usr/src/app/node_modules/ironfish/build/src/blockchain/blockchain.js:569
                    throw new Error(verification.reason);
                          ^

Error: Transaction expired
    at /usr/src/app/node_modules/ironfish/build/src/blockchain/blockchain.js:569:27
    at async LevelupDatabase.withTransaction (/usr/src/app/node_modules/ironfish/build/src/storage/database/database.js:39:28)
    at async Blockchain.newBlock (/usr/src/app/node_modules/ironfish/build/src/blockchain/blockchain.js:519:16)
    at async MiningDirector.constructAndMineBlock (/usr/src/app/node_modules/ironfish/build/src/mining/director.js:257:26)
    at async MiningDirector.constructAndMineBlockWithRetry (/usr/src/app/node_modules/ironfish/build/src/mining/director.js:200:26)
```

**Comment by dguenther** [comment]
This was fixed in 0.1.11 -- Please update your nodes, and if you're still seeing this bug, feel free to open a new issue.
